GameDefinition/Content/ItemKind.hs

```diff
@@ -619,7 +619,7 @@ potion9 = potionTemplate
                    -- has to be weak, or would be too good when thrown at foes
```

Comment:
```suggestion
                   -- the malus has to be weak, or would be too good when thrown at foes
```
(I wasn't clear)

Reply:
Could you commit that bit with message "You do not have permission to push to this repository."? It's buggy and doesn't let me do it (I've just reported the bug).